src/navigation/ApiSidebar.tsx

```diff
@@ -65,66 +142,135 @@ function useMenuItemHook({ tag, idx }) {
+          className="mm-next mm-fullsubopen"
+          // href={id}
+          data-target={states.id}
+          onClick={callbacks.doOpen}
+        >
+          {data.tag}
+        </StyledApiSpan>
+        <div
+          className="mm-panel mm-vertical"
+          id={states.id}
+          style={{
+            marginLeft: "12px",
+            borderLeft: "1px solid #003b45",
+          }}
+        >
+          <ul className="nav-onpage mm-listview mm-vertical">
+            <li>
+              {" "}
+              <StyledApiLink
+                to={statesPath}
+                clicked={
+                  currentHash === stripTrailingSlash("#" + states.anchor)
+                }
+              >
+                {data.tag} Overview
+              </StyledApiLink>
+            </li>
+
+            {/* API menu item child */}
+            {data.endpoints.map((route) => {
+              const routePath = "/api/methods#" + route.anchor;
+              // const currentPage = React.useContext(CurrentPageContext);
+              return (
+                <li key={route.anchor}>
+                  <StyledApiLink
+                    to={routePath}
+                    clicked={
+                      currentHash === stripTrailingSlash("#" + route.anchor)
                     }
-                    style={{
-                      width: "47px",
-                      textAlign: "center",
-                    }}
                   >
-                    {route.httpMethod.toUpperCase()}
-                  </span>{" "}
-                  {route.summary}
-                  {route.tags.includes("Open Endpoint") && (
-                    <OpenEndpointLabel />
-                  )}
-                </Link>
-              </li>
-            );
-          })}
-        </ul>
-      </div>
-    </li>
+                    <MethodDiv>
+                      {route.summary}
+                      <LabelsDiv>
+                        <StyledLabelSpan
+                          className={
+                            "label docs-label-" + route.httpMethod.toLowerCase()
+                          }
+                        >
+                          {route.httpMethod.toUpperCase()}
+                        </StyledLabelSpan>
+                        {route.tags.includes("Open Endpoint") && (
+                          <OpenEndpointLabel />
+                        )}
+                      </LabelsDiv>
+                    </MethodDiv>
+                  </StyledApiLink>
+                </li>
+              );
+            })}
+          </ul>
+        </div>
+      </li>
+    </>
+  );
+}
+
+function ApiMenuItemChildren({
+  endpoints,
+  tag,
+}: {
+  endpoints: EndpointSummary[];
+  tag: string;
+}) {
+  const children = endpoints.map((route) => {
+    const path = "/api/methods#" + route.anchor;
+    // const currentPage = React.useContext(CurrentPageContext);
+    return (
+      // this is one list item (e.g. Account Overview, Delete an Account)
```

Comment:
Comments can be removed, if there's a comment about how the overall function works feel free to add a JSDoc style comment above the function.